fsharp-backend/src/LibBackend/Init.fs

```diff
@@ -55,9 +55,18 @@ let waitForDB () : Task<unit> =
     return ()
   }
 
-
-
-let init (serviceName : string) (runSideEffects : bool) : Task<unit> =
+type WaitForDB =
+  | WaitForDB
+  | DontWaitForDB
+
+/// Initialize LibBackend.
+///
+/// <remarks> This function does not do any behaviour which accesses DB tables and
+/// data, as the DB might not be migrated to it's correct form at this point (eg in
+/// the test of dev environment). This is called by ExecHost, which does the
+/// migration. You cannot expect the DB to be ready when LibBackend is initialized -
+/// call `waitForDB` if you need that.</remarks>
```

Comment:
```suggestion
/// <summary>Initialize LibBackend.</summary>
///
/// <remarks> This function does not do any behaviour which accesses DB tables and
/// data, as the DB might not be migrated to it's correct form at this point (eg in
/// the test of dev environment). This is called by ExecHost, which does the
/// migration. You cannot expect the DB to be ready when LibBackend is initialized -
/// call `waitForDB` if you need that.</remarks>
```

In my experience, a bare `///` is great if you just want "one summary section".

Otherwise, if you want something like `<remarks>`, it's best to wrap each 'section' explicitly (i.e. wrap the beginning here with `<summary>`...`</summary>`. Otherwise, the tool tip you get won't be ideal.

This tends to be true across F# IDEs. Here's the same issue in Rider:
![image](https://user-images.githubusercontent.com/906686/159174539-f9c5d7d3-90e5-4355-9e1e-4a5a64a3fcd3.png)
![image](https://user-images.githubusercontent.com/906686/159174552-a5422338-b98e-4707-9a46-5f2f155395ae.png)

It's minor and silly, but something to keep in mind.

Reply:
Thanks!